learn/user-guide/why-ballerina/network-aware-type-system.md

```diff
@@ -130,13 +130,14 @@ type Result record {|
 |};
 ...
 Result[] results = from var person in persons
-                       let int lgrade = (grades[person.name] ?: 0),
-                       where lgrade > 75
-                           string targetCollege = "Stanford"
-                           select {
-                               name: person.name,
-                               college: targetCollege,
-                               grade: lgrade };
+                  let int lgrade = (grades[person.name] ?: 0), string targetCollege = "Stanford"
+                  where lgrade > 75
+                  select {
+                      name: person.name,
+                      college: targetCollege,
+                      grade: lgrade
+                  };
+
 ```
 
 In the example above, we filter records from a list and create a new value using the `select` clause. Its structure is defined dynamically at that time and the values are created. These values are assigned to an array of `Result` records, which is possible because the generated values are structurally compatible with the `Result` record type.
```

Comment:
```suggestion
In the example above, you filter records from a list and create a new value using the `select` clause. Its structure is defined dynamically at that time and the values are created. These values are assigned to an array of `Result` records, which is possible because the generated values are structurally compatible with the `Result` record type.
```